app/views/expenses/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<p style="color: green"><%= notice %></p>
+
+<div class='expenditure-card'>
+ <div class='expense-heading'>
+    <%= link_to "<<", expenses_path %>
+    <h1>EXPENDITURE</h1>
+  </div>
+
+  <div class='description'>
+     <%= image_tag @expense.icon, alt: @expense.name %>
+     <p><%= @expense.name %></p>
+     
+  </div>
+
+  <% totalAmount = 0 %>
+  <% @expense.expenditures.each do |expenditure| %>
+
+   <div class='expenditure-detail'>
+     <div class='expenditure-id'>
+       <div>
+         <p> Expenditure No:<%= expenditure.id%></p>
+         <p>Amount spent:$<%= expenditure.amount%></p>
+        </div>
+        <%= button_to "Delete", expenditure, method: :delete %>
+      </div>
+      <p><%= expenditure.created_at.strftime('%d %B %Y') %> at <%= expenditure.created_at.strftime('%I:%M %p') %></p>
+    </div>
+    <% totalAmount += expenditure.amount %>
+  <% end %>
+
+  <div class='tot-payment'>
+     <p>TOTAL PAYMENT:</p>
+     <h2>$<%= totalAmount%></h2>
+   </div>
+</div>
+
```

Comment:
- [optional] Great job adding the back-icon taking you back to the expenditure page like the previous reviewer noted, please kindly make it bolder or bigger for a better and more friendly ui/ux design. I advise you get a bigger icon and use there as well. Nice work